objcgen/driver.cs

```diff
@@ -93,7 +93,7 @@ public static int Main2 (params string [] args)
 				Console.WriteLine ($"Embeddinator-4000 v0.1 ({Info.Branch}: {Info.Hash})");
```

Comment:
and it should also be used here, likely need to be done thru `objcgen/Version.generated.cs`